src/applications/personalization/dashboard-2/utils/prescriptions.js

```diff
@@ -0,0 +1,355 @@
+export const prescriptions = {
+  data: [
+    {
+      id: '17927538',
+      type: 'prescriptions',
+      attributes: {
+        prescriptionId: 17927538,
+        prescriptionNumber: '2720040',
+        prescriptionName: 'DAPSONE 25MG TAB',
+        refillStatus: 'submitted',
+        refillSubmitDate: '2021-02-12T15:42:32.000Z',
```

Comment:
So... we are going to want to make all or most of these dates relative to `Date.now()`. Even if we are just using this data for local testing, eventually the data will get too "old" and not be shown since we only show data from the past 30 days. And if we use mock data like this for Cypress we'll have the same kind of issue.

Reply:
Updated!